sprocs/sync_assessments.sql

```diff
@@ -174,6 +175,16 @@ BEGIN
                 syncing_course_instance_id,
                 new_assessment_id
             ) ON CONFLICT DO NOTHING;
+            UPDATE group_configs
+            SET 
+                maximum = (valid_assessment.data->>'group_max_size')::bigint,
+                minimum = (valid_assessment.data->>'group_min_size')::bigint,
+                student_authz_create = (valid_assessment.data->>'group_create')::boolean,
+                student_authz_join = (valid_assessment.data->>'group_join')::boolean,
+                student_authz_leave = (valid_assessment.data->>'group_leave')::boolean
+            WHERE
+                course_instance_id = syncing_course_instance_id
+                AND assessment_id = new_assessment_id;
```

Comment:
The preceding INSERT and this UPDATE would be better as a single UPSERT, I think? In postgres, this is written as
```
INSERT ...
ON CONFLICT DO UPDATE
...
```